app/src/components/blocks/lloydGeorgeRecordDetails/LloydGeorgeRecordDetails.tsx

```diff
@@ -15,6 +15,7 @@ export type Props = {
 
 type PdfActionLink = {
     label: string;
+    dataTestId: string;
```

Comment:
This approach is fine, but maybe if we can change `dataTestId` to a name that can be reusable as these are technically unique identifies. 

Maybe change the prop to be called `key` or `id` instead. Then we can share this prop on the key and data-testid part of the mapped link render instead of the concatenated (and ugly) key currently on the links

Reply:
amended